### PR TITLE
Median of two arrays in python 

### LINKED ISSUE #5373: Median of two arrays in python
## 💥 Proposal

addition of median of two arrays in python. I am a GSSOC participant and want to contribute in python.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

yes

### PR DESCRIPTION
**Pull request by Amit366**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

yes

### Description

Added Median of arrays in python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes: #5373

### CODE REVIEW

**Comment by rohanbabbar04** [comment]
@Amit366 Go to deepsource:Python and correct all the errors

**Comment by Amit366** [comment]
@HarshCasper please review.

File: Python/cp/Median_of_two_sorted_arrays.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        nums1.append(x)
+
+    print('Enter the elements of second array: ')
+    for i in range(m):
+        y = int(input())
+        nums2.append(y)
+
+    print('Median of two arrays is: ' +
+          str(findMedianSortedArrays(nums1, nums2)))
+
```
Comment (HarshCasper) [inline Python/cp/Median_of_two_sorted_arrays.py:32]: Add a Driver Code

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/cp/Median_of_two_sorted_arrays.py (hunk 1 above)
Comment (Amit366) [reply Python/cp/Median_of_two_sorted_arrays.py:32]: Means main??

**Comment by Amit366** [comment]
@HarshCasper done

**Comment by Amit366** [comment]
@HarshCasper please review

File: Python/cp/Median_of_two_sorted_arrays.py (hunk 1 above)
Comment (HarshCasper) [reply Python/cp/Median_of_two_sorted_arrays.py:32]: Yes

**Comment by Amit366** [comment]
@HarshCasper is it fine ??

File: Python/cp/Median_of_two_sorted_arrays.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    # creating an empty lists
+    nums1 = []
+    nums2 = []
+    n = int(input('Enter length of first array: '))
+    m = int(input('Enter length of second array: '))
+
+    print('Enter the elements of first array: ')
+    for i in range(n):
+        x = int(input())
+        nums1.append(x)
```
Comment (rohanbabbar04) [inline Python/cp/Median_of_two_sorted_arrays.py]: Use list map

File: Python/cp/Median_of_two_sorted_arrays.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    print('Enter the elements of second array: ')
+    for i in range(m):
+        y = int(input())
+        nums2.append(y)
```
Comment (rohanbabbar04) [inline Python/cp/Median_of_two_sorted_arrays.py]: Use list map instead of a for loop

**Review by rohanbabbar04 — CHANGES_REQUESTED** [review]

File: Python/cp/Median_of_two_sorted_arrays.py (hunk 3 above)
Comment (Amit366) [reply Python/cp/Median_of_two_sorted_arrays.py]: @rohanbabbar04 can help me? I am not getting how to change.

Comment (rohanbabbar04) [reply Python/cp/Median_of_two_sorted_arrays.py]: @Amit366 refer this https://www.geeksforgeeks.org/python-get-a-list-as-input-from-user/ 
Read code number 3

Comment (Amit366) [reply Python/cp/Median_of_two_sorted_arrays.py]: @rohanbabbar04 done

**Comment by Amit366** [comment]
@HarshCasper @ankitaggarwal23 please review

**Review by HarshCasper — APPROVED** [review]

**Comment by Amit366** [comment]
@HarshCasper please add the labels.

**Pull request merged**